pages/index.tsx

```diff
@@ -71,8 +71,8 @@ export const Index: NextPage = () => {
 
   return (
     <PageLayout
-      siteTitle='homepage.siteTitle'
-      siteDescription='homepage.siteDescription'
+      pageTitle='homepage.pageTitle'
```

Comment:
shouldn't this be `pageTitle={t('homepage.pageTitle')}`?

Reply:
I don't know. (I didn't change it.)

Reply:
sometimes we pass title and description has a translation object (`homepage.siteDescription`) and some time as translated string (`t('homepage.siteDescription')`). 
I don't understand how they can both work.. 🤔

Reply:
Please find out. But let me merge this! 😅

Reply:
i also want to merge it but i also want to know it works. I know you didn't change it but better we check before we discover weird behaviours later on, no?

Reply:
I push a commit